src/Controls/src/Build.Tasks/nuget/buildTransitive/net6.0-maccatalyst13.1/Microsoft.Maui.Controls.DefaultItems.targets

```diff
@@ -0,0 +1,17 @@
+<Project>
+
+	<PropertyGroup>
+		<EnableDefaultMauiBlazorEntitlements Condition="'$(EnableDefaultMauiBlazorEntitlements)'=='' And '$(UsingMicrosoftNETSdkRazor)'=='true'">true</EnableDefaultMauiBlazorEntitlements>
```

Comment:
Hmm this essentially looks like it's hard-coding some Blazor Hybrid logic into this target, which seems a bit too specific. I'm not sure this logic is quite right, either, because I don't think the MAUI project itself has to necessarily use the Razor SDK, yet the app likely still wants to use these entitlements if it has Blazor usage in it (perhaps indirectly through a project reference).

Reply:
Hmm, I got the idea from the [Microsoft.Maui.Sdk.ProjectCapabilities.targets](https://github.com/dotnet/maui/blob/186ba51c1fca86f06c36aadd15f11f7f24e4490c/src/Workload/Microsoft.Maui.Sdk/Sdk/Microsoft.Maui.Sdk.ProjectCapabilities.targets#L4) file. It would have been more favorable to use the MauiBlazor ProjectCapability directly, but the condition checking for that wasn't working in the integration test.

I had not thought of an indirect reference without using the Razor SDK though. I will investigate and see if I can come up with an alternative means.

Reply:
Oh weird... I don't really like that either. Looks like it's ancient. I think it's a reasonable for _telemetry_ but I'm not sure I like it for _functional_ purposes... Though it doesn't even necessarily look right for that because it doesn't actually check if it's MAUI SDK + Razor, though probably that's fine because this very target is from MAUI to begin with.